Quadriga/src/main/java/edu/asu/spring/quadriga/service/network/impl/NetworkManager.java

```diff
@@ -455,6 +466,28 @@ public String storeNetworkDetails(String xml, IUser user, String networkName, St
     }
 
     /**
+     * {@inheritDoc}
+     */
+    @Override
+    public List<CreationEvent> getTopElementEvents(String xml) throws JAXBException {
+        ElementEventsType elementEventType = marshallingService.unMarshalXmlToElementEventsType(xml);
+        NewNetworkDetailsCache newNetworkDetailCache = new NewNetworkDetailsCache();
```

Comment:
give that you replace the new instance in the next line, the instantiation is unnecessary.